src/main/java/com/google/sps/servlets/MatchServlet.java

```diff
@@ -49,8 +65,24 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
         //return null for matches so that page redirects to logged out homepage
         response.getWriter().println(gson.toJson(null));
     } else {
-        Collection<User> matches = matchRepository.getMatchesForUser(sessionContext.getLoggedInUser());
-        response.getWriter().println(gson.toJson(matches));
+        // calling analyze request 
```

Comment:
L68: Though I'm all for more comments, they only should be used if what they convey is non-obvious; I can see from the code (`boolean value = analyzeRequest(request);`) that it's calling analyzeRequest; What I don't know is what analyzeRequest is supposed to do (largely because "analyzeRequest" isn't particularly descriptive).

Reply:
Done